BusinessLogic/Services/SettingService.cs

```diff
@@ -17,11 +17,26 @@ public void Create(Settings settings)
             settingsRepository.Create(settings);
         }
 
-        public void Update(Settings settings)
+        public void UpdateSetting(Settings settings)
         {
             settingsRepository.Update(settings);
-          
-            
+        }
+
+        public bool ValidateColor(Classes.Color convertedColor)
+        {
+            //not empty
+            if (convertedColor == null)
+                return false;
+
+            if (convertedColor.Alpha <= 0)
```

Comment:
Nu is een alpha van 1 ook acceptable, denk dat we eerder gewoon moeten zeggen dat het 255 moet zijn?